frontend/style/main.css

```
@@ -869,3 +869,37 @@ textarea:focus-visible {
+  color: var(--color-text);
+  border-bottom: 1px solid var(--color-error);
+  font-size: 0.875rem;
+}
+
+.history-error-text {
+  flex: 1;
+}
+
+.history-error-dismiss {
+  padding: 0.25rem 0.5rem;
+  border: 1px solid currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)